vendor/bat-native-ads/src/bat/ads/internal/account/ad_rewards/ad_rewards.cc

```diff
@@ -324,7 +325,8 @@ void AdRewards::OnGetAdGrants(const UrlResponse& url_response) {
   }
 
   if (!ad_grants_->SetFromJson(url_response.body)) {
-    BLOG(0, "Failed to parse ad grants: " << url_response.body);
+    BLOG(0, "Failed to parse ad grants");
+    BLOG(1, "Ad grants response body: " << url_response.body);
```

Comment:
Elsewhere in ads and ledger we have used `6` for responses, would be good to use the same value so that these are only shown when the verbosity is 6 or higher.